sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIO.java

```diff
@@ -432,6 +435,8 @@ BigtableService getBigtableService(PipelineOptions pipelineOptions) {
     @Nullable
```

Comment:
Please add tests that make sure withoutValidation() actually disables validation.

Reply:
👍